cypress/integration/00:auth/1:login.feature

```diff
@@ -0,0 +1,32 @@
+@other
+Feature: LOGIN: Login user
+  This feature allows login user
+
+  Background:
+    Given I open "" page
+
+  @other
+  Scenario: Login user - validation error (misc string)
+    When I fill the "login-pass" input with the "password" term
+    * I click "show-password" check
+    * On "login-pass" input I can see "password" term
+
+  @error
+  Scenario: Login user - validation error (misc string)
+    When I fill the "login-email" input with the "login" term
+    * I fill the "login-pass" input with the "password" term
+    * I "submit" the data and "create" is "incorrect"
+    * I see a form validation error that says "['Invalid credentials.']"
+
+  @error
+  Scenario: Login user - validation error (empty string)
+    When I click on "submit" button
+    * I see a form validation error that says "['E-mail is required','Password is required']"
+
+  @success
+  Scenario: Login user - success
+    When On login page I fill the "login-email" input with the "adminEmail" term
```

Comment:
Why not using the already defined quote?  I fill the "X" input with the "Y" term

Reply:
because in this context I am using the login data taken from the .env variables so I had to overwrite it in this space